t/ui/23-audit-log.t

```diff
@@ -67,42 +71,45 @@ ok($search, 'search box found');
 my @entries = $driver->find_child_elements($table, 'tbody/tr');
 is(scalar @entries, 3, 'three elements without filter');
 
+$search->send_keys('QA restart');
+wait_for_data_table;
+@entries = $driver->find_child_elements($table, 'tbody/tr');
+is(scalar @entries, 1, 'one element when filtered for event data');
+like($entries[0]->get_text(), qr/openQA restarted/, 'correct element displayed');
+$search->clear;
+wait_for_data_table;
+
 $search->send_keys('user:system');
-t::ui::PhantomTest::wait_for_ajax;
-sleep 1;
+wait_for_data_table;
 @entries = $driver->find_child_elements($table, 'tbody/tr');
 is(scalar @entries, 1, 'one element when filtered by user');
 $search->clear;
-t::ui::PhantomTest::wait_for_ajax;
+wait_for_data_table;
 
 $search->send_keys('event:user_login');
-t::ui::PhantomTest::wait_for_ajax;
-sleep 1;
+wait_for_data_table;
 @entries = $driver->find_child_elements($table, 'tbody/tr');
 is(scalar @entries, 2, 'two elements when filtered by event');
 $search->clear;
-t::ui::PhantomTest::wait_for_ajax;
+wait_for_data_table;
 
 $search->send_keys('newer:today');
-t::ui::PhantomTest::wait_for_ajax;
-sleep 1;
+wait_for_data_table;
 @entries = $driver->find_child_elements($table, 'tbody/tr');
 is(scalar @entries, 3, 'three elements when filtered by today time');
 $search->clear;
-t::ui::PhantomTest::wait_for_ajax;
+wait_for_data_table;
 
 $search->send_keys('older:today');
-t::ui::PhantomTest::wait_for_ajax;
-sleep 1;
+wait_for_data_table;
 @entries = $driver->find_child_elements($table, 'tbody/tr/td');
 is(scalar @entries, 1, 'one element when filtered by yesterday time');
 is($entries[0]->get_attribute('class'), 'dataTables_empty', 'but datatables are empty');
 $search->clear;
-t::ui::PhantomTest::wait_for_ajax;
+wait_for_data_table;
```

Comment:
why do you also replace the wait_for_ajax calls which apparently worked with the `wait_for_table_data` which is just a `sleep 1` for now? Better just replace only the `sleep 1` with this method and keep the `wait_for_ajax` in as is